packages/module-backup/src/client/Configuration.tsx

```
@@ -299,18 +318,32 @@ const NewBackup = ({ ButtonComponent = Button, refresh }) => {
-          <Button key="cancel" onClick={handleCancel}>
-            {t('Cancel')}
-          </Button>,
-          // 明确使用工作线程备份
-          <Button key="custom" type="primary" onClick={() => handleOk('worker')}>
-            {t('Worker backup')}
-          </Button>,
-          // 明确使用自身线程备份
-          <Button key="ok" onClick={() => handleOk('main')}>
-            {t('Self backup')}
-          </Button>,
+          <Row gutter={16} justify="end" align="middle">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)